### ISSUE TITLE
Logfile reports incorrect finish date

### ISSUE THREAD

**Issue by spadavec** [choderalab/yank#713]
Sorry for pestering y'all with silly issues, but it looks like repex.py is reporting the wrong estimated finish date (its off by about 6 months). 

From my log file: 
```
2017-07-11 07:29:49,165: Estimated completion in 1 day, 11:53:50.242195, at Sat Jan  3 15:13:29 1970 (consuming total wall clock time 2 days, 23:13:29.210704).
2017-07-11 07:29:49,166: Iteration 497/1000
```
The wall clock time is correct, as is the estimated completion time, but it looks like the 'partial_total_time` (that contributes to end date prediction) is wrong in: 

>             if logger.isEnabledFor(logging.DEBUG):
>                 iteration_time = timer.stop('Iteration')
>                 partial_total_time = timer.partial('Run ReplicaExchange')
>                 time_per_iteration = partial_total_time / (self._iteration - run_initial_iteration)
>                 estimated_time_remaining = time_per_iteration * (iteration_limit - self._iteration)
>                 estimated_total_time = time_per_iteration * iteration_limit
>                 estimated_finish_time = partial_total_time + estimated_time_remaining
>                 logger.debug("Iteration took {:.3f}s.".format(iteration_time))
>                 logger.debug("Estimated completion in {}, at {} (consuming total wall clock time {}).".format(
>                     str(datetime.timedelta(seconds=estimated_time_remaining)), time.ctime(estimated_finish_time),
>                     str(datetime.timedelta(seconds=estimated_total_time))))

**Comment by jchodera** [comment]
> at Sat Jan  3 15:13:29 1970

Whoops! The suggestion that we've invented time travel is a bit embarrassing.

@Lnaden : Can you fix?

**Comment by Lnaden** [comment]
> The suggestion that we've invented time travel is a bit embarrassing.

Oh no! You found my secret back channel to the past where I have been quietly investing small quantities that equal millions in today's money! <end joke>

Yeah, I'll fix that with the 0.17.0 pr I have open.
